components/quickbooks/actions/sandbox-search-invoices/sandbox-search-invoices.mjs

```diff
@@ -1,84 +1,86 @@
-// legacy_hash_id: a_OOiLgK
-import { axios } from "@pipedream/platform";
+import { ConfigurationError } from "@pipedream/platform";
+import quickbooks from "../../quickbooks.app.mjs";
 
 export default {
   key: "quickbooks-sandbox-search-invoices",
   name: "Search Invoices",
-  description: "Searches for invoices.",
-  version: "0.1.1",
+  description: "Searches for invoices. [See docs here](https://developer.intuit.com/app/developer/qbo/docs/api/accounting/all-entities/invoice#query-an-invoice)",
+  version: "0.1.2",
   type: "action",
   props: {
-    quickbooks: {
-      type: "app",
-      app: "quickbooks",
-    },
-    include_clause: {
-      type: "string",
+    quickbooks,
+    includeClause: {
```

Comment:
Can we move `includeClause` to the app file? It is used in all querying actions.